extensions/librdkafka/PublishKafka.cpp

```diff
@@ -99,6 +99,14 @@ core::Property PublishKafka::MessageKeyField("Message Key Field", "The name of a
                                              "");
 core::Property PublishKafka::DebugContexts("Debug contexts", "A comma-separated list of debug contexts to enable."
                                            "Including: generic, broker, topic, metadata, feature, queue, msg, protocol, cgrp, security, fetch, interceptor, plugin, consumer, admin, eos, all", "");
+const core::Property PublishKafka::DropEmptyFlowFiles(
+    core::PropertyBuilder::createProperty("Drop empty flow files")
+        ->withDescription("Keep backwards compatibility with <=0.7.0 bug which caused flow files with empty content to not be published to Kafka. The old behavior is deprecated. Use connections "
+                          "to drop empty flow files!")
+        ->isRequired(false)
+        ->withDefaultValue<bool>(true)
+        ->build()
+);
```

Comment:
Just nitpicking, but it's usually at the end of the last call (usually "build")

Reply:
What is at the end of the last call? Do you mean the closing paren and semicolon?

Reply:
Yep

Reply:
fixed in c872f3d